lib/mobility/backends/active_record/key_value.rb

```diff
@@ -204,12 +211,15 @@ def self.included(model_class)
           model_class.after_destroy do
             [:string, :text].each do |type|
               Mobility::Backends::ActiveRecord::KeyValue.const_get("#{type.capitalize}Translation").
-                where(translatable: self).destroy_all
+                where(translatable => self).destroy_all
             end
           end
         end
       end
 
+      # FIXME: replace:
+      #   - `:translatable` with `options[:translatable]`
+      #   - `:key` with `options[:key_column]`
```

Comment:
With my limited Ruby-fu I don't know how to declare this class in a way that respects the :translatable and :key_column options I added.

Reply:
In order to achieve this, we'd need to make this class and its subclasses dynamically generated in the `setup` block. This is possible and shouldn't be a problem since we're anyway subclassing these classes now.

Reply:
This shouldn't be necessary, since we're setting the `translation_class` to `::ActionText::RichText` (see my commits) and we thus never use this translation class.